source/MaterialXCore/Document.cpp

```diff
@@ -305,6 +307,30 @@ ValuePtr Document::getGeomPropValue(const string& geomPropName, const string& ge
     return value;
 }
 
+std::vector<OutputPtr> Document::getMaterialOutputs(bool skipIncludes) const
```

Comment:
Is there any valid case where we _wouldn't_ want to skip XIncluded content?  As a user, I would only expect this method to find material outputs in the content file, not in the underlying data libraries.

Reply:
I wasn't sure if it's always considered that an `xinclude` references a "library" not or if the author just wanted to split up the content file into multiple files. For simplicity I will remove this argument for now. (Adding an argument more backward compatible than removing one.)